src/Backend/Modules/ContentBlocks/Installer/Installer.php

```diff
@@ -20,34 +20,43 @@ class Installer extends ModuleInstaller
 {
     public function install(): void
     {
-        // add the schema of the entity to the database
-        Model::get('fork.entity.create_schema')->forEntityClass(ContentBlock::class);
-
-        // add 'content_blocks' as a module
         $this->addModule('ContentBlocks');
-
-        // import locale
         $this->importLocale(__DIR__ . '/Data/locale.xml');
+        $this->configureEntities();
+        $this->configureSettings();
+        $this->configureBackendNavigation();
+        $this->configureBackendRights();
+    }
 
-        // general settings
-        $this->setSetting($this->getModule(), 'max_num_revisions', 20);
+    private function configureBackendNavigation(): void
+    {
+        // Set navigation for "Modules"
```

Comment:
I think this comment can be removed since it is rather vague and the method name already explains it